<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Biblioteca </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white; background-color: #333333; padding: 10px; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>LISTA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>601</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #333333;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>601</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="botao_voltar">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white; border-radius: 5px;
background-color: rgb(170, 0, 255);</string>
        </property>
        <property name="text">
         <string>Voltar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botao_disponiveis">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> color: white; border-radius: 5px;
background-color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>Dispon√≠veis</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botao_meusLivros">
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> color: white; border-radius: 5px;
background-color: rgb(117, 117, 117);</string>
        </property>
        <property name="text">
         <string>Meus Livros</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>verticalLayoutWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
